### PR TITLE
Add brief bio and picture

### PR DESCRIPTION
**Pull request by ninaprabhu**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,77 @@
...
+          funny shows like How I Met Your Mother.</p>
+      <img src="/images/stanford.jpg" />
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      
+    </div>
+
+    <div id = "professional">
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:24]: Nit: remove extra spaces

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,77 @@
...
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      
+    </div>
+
+    <div id = "professional"> 
+        <p> You can see my full work and background on my  <a href="https://github.com/ninaprabhu">Github</a> and my <a href="https://www.linkedin.com/in/nina-prabhu">LinkedIn</a>.
+        Languages: Python, C++, C, SAS, HTML/CSS
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:26]: Missing closing </p>

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,77 @@
...
+      <img src="/images/stanford.jpg" />
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+
+      
+    </div>
+
+    <div id = "professional"> 
+        <p> You can see my full work and background on my  <a href="https://github.com/ninaprabhu">Github</a> and my <a href="https://www.linkedin.com/in/nina-prabhu">LinkedIn</a>.
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:25]: Remove spaces after tags for consistency. (This isn't explicitly stated in Google's style guide but it's pretty standard to not have a space here.)

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,77 @@
...
+        Languages: Python, C++, C, SAS, HTML/CSS 
+
+        <p> Below are some my favorite projects that I've worked on. </p>
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:28]: Trailing whitespace can be problematic: https://google.github.io/styleguide/htmlcssguide.html#Trailing_Whitespace

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -2,17 +2,77 @@
...
+                        
+                    <li> To share this and other findings, I co-authored my team's paper that was presented at the 
+                        AAAI 2019 conference. </li>
+                </ul>
+
+            </li>
+            
+        </ul>
+
+        <!-- Project template
```
Comment (Tetheta) [inline portfolio/src/main/webapp/index.html:65]: Remove commented code unless it's truly necessary for some reason.

Comment (ninaprabhu) [reply portfolio/src/main/webapp/index.html]: Removed! I made another commit to this branch that should reflect this.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (ninaprabhu) [reply portfolio/src/main/webapp/index.html]: I removed extra spaces and added the ending </p> tag! I made another commit to this branch that should reflect this.

**Pull request merged**